scripts/lib/install

```diff
@@ -175,16 +175,30 @@ export LANGUAGE="en_US.UTF-8"
 
 # Check for a supported OS release.
 if [ -f /etc/os-release ]; then
+    os_info_check="$(
+        . /etc/os-release
+        printf '%s\n' "$PRETTY_NAME"
+    )"
     os_info="$(
         . /etc/os-release
         printf '%s\n' "$ID" "$ID_LIKE" "$VERSION_ID" "$VERSION_CODENAME"
     )"
     {
-        read -r os_id
-        read -r os_id_like
-        read -r os_version_id
-        read -r os_version_codename || true
-    } <<<"$os_info"
+        read -r os_pretty_name
+    } <<<"$os_info_check"
+    if [ "$os_pretty_name" = "Debian GNU/Linux bullseye/sid" ]; then
+        os_id="debian"
+        os_id_like="debian"
+        os_version_id="11"
+        os_version_codename="bullseye"
+    else
+        {
+            read -r os_id
+            read -r os_id_like
+            read -r os_version_id
+            read -r os_version_codename || true
+        } <<<"$os_info"
+    fi
```

Comment:
This block of logic is pretty complicated, and is also repeated in various forms in like 3 files.  I don't want to merge this sort of complex restructuring, which we expect to not be relevant until Buster is actually released.

Here's an alternative proposal: We could just modify `tools/ci/production-install` to at its start, for buster, overwrite `/etc/os-release` with what we expect its contents to be once Buster is released, and then we can leave all of our parsing logic unmodified.

Reply:
I guess you are referring to `bullseye` instead of `buster` but I got the gist of it.